packages/api/src/routes/beacon/index.ts

```diff
@@ -1,9 +1,9 @@
 import {IChainForkConfig} from "@chainsafe/lodestar-config";
 import {phase0, ssz} from "@chainsafe/lodestar-types";
-import {RoutesData, ReturnTypes, reqEmpty, ContainerData} from "../../utils";
 import * as block from "./block";
 import * as pool from "./pool";
 import * as state from "./state";
+import {RoutesData, ReturnTypes, reqEmpty, ContainerData} from "../../utils";
```

Comment:
So we actually want the closest modules at the bottom, so the correct order would be the original one. @dadepo can you try to tweak the rule to achieve that?

Reply:
Updated!